style.css

```diff
@@ -0,0 +1,93 @@
+/* GLOBAL RULES */
+
+:root {
+  --main-color: #172b4d;
+  --secondary-color: #36b37e;
```

Comment:
if you didn't use this color you can delete it and take care of the color contrast